### PR TITLE
Enterprise Support

### PR DESCRIPTION
**Pull request by benbjohnson**
## Overview

Adds `Key` fields to support translation.

### CODE REVIEW

File: orm.go [hunk 1]
```diff
@@ -303,6 +303,11 @@ func (idx *Index) Name() string {
 	return idx.name
 }
 
+// ColumnLabel returns the column label for this index.
+func (idx *Index) ColumnLabel() string {
```
Comment (yuce) [inline orm.go]: column/row labels were deprecated in the server and clients and removed in the cluster-resize compatibility PR at: https://github.com/pilosa/go-pilosa/pull/91 (to be merged when the cluster-resize branch is merged.) Do we need support for them in the enterprise server ?

File: orm.go [hunk 2]
```diff
@@ -573,6 +578,11 @@ func (f *Frame) Name() string {
 	return f.name
 }
 
+// RowLabel returns the row label for this frame.
+func (f *Frame) RowLabel() string {
```
Comment (yuce) [inline orm.go]: (the same remark with column labels)

File: orm.go (hunk 1 above)
Comment (jaffee) [reply orm.go]: I don't have any issue with dropping them.

**Comment by benbjohnson** [comment]
Closing. Use #95 instead.

**Pull request closed without merging**